agent.py

```
@@ -30,9 +32,8 @@
 )
 
 from dotenv import load_dotenv
-import asyncio # Added asyncio
-from pathlib import Path # Added Path
-from config import set_constant, get_constant, MAIN_MODEL # Added config imports
+from pathlib import Path
+from config import set_constant, get_constant, MAIN_MODEL
```

Ruff (0.11.9)

Redefinition of unused `MAIN_MODEL` from line 29

Remove definition: `MAIN_MODEL`

(F811)